clients/client-shell/cmds/group/actors.go

```diff
@@ -21,7 +21,12 @@ func init() {
 	cancelCmd := &cobra.Command{
 		Use:   "cancel <taskGroupId>",
 		Short: "Cancel a whole group by taskGroupId.",
-		RunE:  executeHelperE(runCancel),
+		Long: "This method fetches all tasks in the given task group, and cancels\n" +
+			"all tasks with respect to worker-type filter if given.\n\n" +
+			"Note that there is a more efficient way to cancel a task group with API:\n\n" +
+			"taskcluster api queue sealTaskGroup <taskGroupId> # seal task group \n" +
```

Comment:
I feel like we should mention how this step is _required_ for being able to cancel the task group. Maybe something like this?
```suggestion
			"taskcluster api queue sealTaskGroup <taskGroupId> # seal task group is required before calling cancelTaskGroup\n" +
```

Reply:
Thanks! WIll add